src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -39,17 +39,29 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
       .getOrElse(dataManager.getPath(0))
   }
 
+  protected def getNumBucketsConfig(spark: SparkSession): Int = {
+    spark.sessionState.conf
```

Comment:
Can we move this to `HyperspaceConf`?

Reply:
And we can call it `numBuckets` and `lineageEnabled` respectively. (or `numBucketsForIndex` and `indexLineageEnabled`)